octopoes/octopoes/api/api.py

```diff
@@ -57,47 +58,56 @@
 
 
 @app.exception_handler(RequestValidationError)
-def http_validation_exception_handler(request: Request, exc: RequestValidationError) -> JSONResponse:
-    logger.critical(exc)
-    return JSONResponse(
-        {
-            "value": str(exc),
-        },
-        status.HTTP_422_UNPROCESSABLE_ENTITY,
-    )
+def http_validation_exception_handler(_: Request, exc: RequestValidationError) -> None:
+    logger.info(exc)
+    raise HTTPException(status.HTTP_422_UNPROCESSABLE_ENTITY, detail=str(exc))
 
 
 @app.exception_handler(RequestError)
-def http_exception_handler(request: Request, exc: RequestError) -> JSONResponse:
-    logger.critical(exc)
-    return JSONResponse(
-        {
-            "value": str(exc),
-        },
-        status.HTTP_502_BAD_GATEWAY,
-    )
+def http_exception_handler(_: Request, exc: RequestError) -> None:
+    logger.error(exc)
+    raise HTTPException(status.HTTP_502_BAD_GATEWAY, detail=str(exc))
```

Comment:
This was already the case, but this should be 500 and not 502 because Octopoes is not acting as a gateway.